### ISSUE TITLE
issues with scanning and verifying between two versions

### ISSUE THREAD

**Issue by ubuntusuperuser** [speed47/dvdisaster#65]
Created a disk using version 0.79.5 (from the default repos in ubuntu 22.04 LTS)

scaning the disk and verifying the .iso file works fine on version 0.79.6 on windows 10

![0 79 6 scan](https://user-images.githubusercontent.com/7614894/224466157-c2154fd5-1b9e-4e8f-afb1-fa0cde2bb8e3.png)
![0 79 6 verify](https://user-images.githubusercontent.com/7614894/224466161-6030c210-c26e-4b0d-8842-74be7ee6bf9c.png)

However using version 0.79.10 on windows 10 gives errors
![0 79 10 scan](https://user-images.githubusercontent.com/7614894/224466164-597db833-ed86-4141-9627-e015a37b462a.png)
![0 79 10 verify](https://user-images.githubusercontent.com/7614894/224466165-74688210-b5fa-4d9c-ad27-d9f1bbdefab0.png)

**Comment by speed47** [comment]
Hello, thanks for your report.

Is the behaviour on Windows 10 with 0.79.10p1 reproducible, i.e. if you scan it again, you still get errors? If yes, do you get exactly the same amount of errors or does it differ?

You're saying it works on Windows 10 with 0.79.6p1, could you try with the last 0.79.6 patchlevel before the 0.79.10 version, i.e. this one? [v0.79.6 patchlevel 9](https://github.com/speed47/dvdisaster/releases/tag/v0.79.6-pl9)

**Comment by ubuntusuperuser** [comment]
yes it is reproducible on windows 10 0.79.10p1, consistantly the same amount of errors.
0.79.6p9 on windows works with no errors.
I just tried using 0.79.10p1 on ubuntu using the appimage and that works with no errors

**Comment by Techrocket9** [comment]
On Windows 11 with 0.79.10 I'm getting CRC errors on all non-ECC blocks when I augment an ISO.
![CRC](https://github.com/speed47/dvdisaster/assets/611505/990509b1-4c2b-4908-9b50-e159c146e7d5)
 
 The contents of the ISO pass a "7zip test archive", but dvdisaster doesn't seem to be able to parse it properly.

**Comment by speed47** [comment]
I believe this is fixed in the latest version: 

- fix: corrupt data under Windows for RS03 when memory-mapped i/o strategy was used (https://github.com/speed47/dvdisaster/issues/82)

Can you retry with the latest version to confirm this?
